Update initial usage example (#16824)
docker/docs: desktop/wasm/index.md

  michaelirwin244/wasm-example
```

> After running this command, you should be able to visit http://localhost:8080/ and see the "Hello world" output from this example module.

Reviewer: ```suggestion
After running this command, you can visit [http://localhost:8080/](http://localhost:8080/) to see the "Hello world" output from this example module.
```
